LadybugTools_oM/ExecuteCommands/RunHeatPlotCommand.cs

```diff
@@ -0,0 +1,24 @@
+﻿using BH.oM.Adapter;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+
+namespace BH.oM.LadybugTools
+{
+    [Description("Command that, when executed with the LadybugToolsAdapter, produces a heatmap from data in an epw file.\nOutput is a string of either the path to the image (if OutputLocation is not set) or the base 64 string representation of it.")]
+    public class RunHeatPlotCommand : IExecuteCommand
+    {
+        [Description("The path to an EPW file.")]
+        public virtual string EpwFile { get; set; } = "";
+
+        [Description("Key representing an hourly continuous collection in the EPW file to plot.")]
+        public virtual EpwKey EpwKey { get; set; } = EpwKey.Undefined;
+
+        [Description("A Matplotlib colout map. Corresponds to the 'cmap' parameter of plot methods. See https://matplotlib.org/stable/users/explain/colors/colormaps.html for examples of valid keys. Default of 'viridis'.")]
+        public virtual string ColourMap { get; set; } = "viridis";
```

Comment:
See other comment on this being an enum.

Reply:
Let's take out the colour map functionality for now. This is going to need a lot more thought.